backend/src/libs/connection.js

```diff
@@ -1,35 +1,20 @@
 import mariadb from 'mariadb';
 
-export const DB_CONNECTION_KEY = 'dbConnection';
-
-const pool = mariadb.createPool({
-  host: 'localhost',
-  port: 3306,
-  user: 'quackerUser',
-  password: 'quackerPassword',
-  database: 'quacker',
-  connectionLimit: 5,
-});
-
-const getConnection = async () => {
+export const getConnection = async () => {
   let conn;
   try {
-    conn = await pool.getConnection();
+    conn = await mariadb.createConnection(
+      {
+        host: 'localhost',
```

Comment:
Please add `process.env.DB_HOST` with default value of `DB_HOST=localhost`.

It may be possible that some students will have DB on some server for local development.